test/blocks/columns/columns.plain.html

```diff
@@ -0,0 +1,22 @@
+  <div>
+    <div class="columns split-80-20 native-image-sizes">
+      <div>
+        <div id="column-a" data-valign="middle">
```

Comment:
Here a,b,x,y seem to be siblings. I thought a,b and x,y should be siblings with 80-20 rule being applied to each sibling pair.

Reply:
Yes there was something incorrect in the test .html resource. I fixed that now, well spotted!